pkg/auth/metadata.go

```diff
@@ -268,5 +268,5 @@ func (m *KVMetadataManager) GetMetadata(ctx context.Context) (map[string]string,
 		metadata[SetupTimestampKeyName] = setupTS.UTC().Format(time.RFC3339)
 	}
 
-	return metadata, err
+	return metadata, nil
```

Comment:
No need to return nil instead of err if you are already not failing for it when calling this function later

Reply:
true, but I wanted to note that this is ok, as not all metadata is collected